### ISSUE TITLE
EDB API: deactivate_rlc_component doesn't make a proper gap port.

### ISSUE THREAD

**Issue by askoeho** [ansys/pyaedt#3838]
### Before submitting the issue

- [X] I have searched among the existing issues
- [X] I am using a Python virtual environment

### Description of the bug

As far as I have understood correctly, the following line of code should make a gap port for a RLC component. But for some reason that is not working.

`edb.components.deactivate_rlc_component(component=smd, create_circuit_port=False)`

### Steps To Reproduce

`edb.components.deactivate_rlc_component(component=smd, create_circuit_port=False)`

-> No gap ports are generated.

### Which Operating System are you using?

Windows

### Which Python version are you using?

3.10

### Installed packages

accelerate==0.23.0
adodbapi==2.6.1.3
affine==2.3.1
aiofiles==22.1.0
aiohttp==3.8.3
aiosignal==1.2.0
aiosqlite==0.17.0
alabaster==0.7.12
alembic==1.11.1
algopy==0.5.7
altair==5.1.1
altair-data-server==0.4.1
altair-transform==0.2.0
aniso8601==9.0.1
annotated-types==0.5.0
ansi2html==1.6.0
ansiwrap==0.8.4
ansys-pythonnet==3.1.0rc3
anyio==3.7.1
appdirs==1.4.4
argon2-cffi==21.3.0
argon2-cffi-bindings==21.2.0
arrow==1.2.2
arviz==0.15.1
asciitree==0.3.3
asgi-csrf==0.9
asgiref==3.5.2
asn1crypto==1.4.0
asteval==0.9.27
astroid==2.12.12
astropy==5.3.2
asttokens==2.1.0
async-lru==2.0.4
async-timeout==4.0.3
atomicwrites==1.4.0
attrs==22.1.0
autopep8==1.7.0
azure-core==1.29.2
azure-cosmos==4.5.0
azure-identity==1.14.0
Babel==2.10.3
backcall==0.2.0
baresql==0.8.0
bcrypt==3.2.0
beautifulsoup4==4.11.1
binaryornot==0.4.4
black==23.9.1
bleach==6.0.0
blinker==1.4
blis==0.7.10
blosc==1.10.6
bokeh==3.2.2
botorch==0.8.5
Bottleneck==1.3.4
bqplot==0.12.40
branca==0.6.0
brewer2mpl==1.4.1
Brotli==1.0.9
build==0.10.0
CacheControl==0.12.11
cachelib==0.9.0
cachetools==5.2.0
catalogue==2.0.8
certifi==2022.9.24
cffi==1.15.0
cftime==1.6.0
chardet==5.0.0
charset-normalizer==2.0.12
cleo==2.0.1
click==8.1.3
click-default-group==1.2.4
click-default-group-wheel==1.2.2
click-plugins==1.1.1
cligj==0.7.2
cloudpickle==2.1.0
clr-loader==0.2.6
clrmagic==0.0.1a2
colorama==0.4.6
colorcet==3.0.1
comm==0.1.3
confection==0.1.1
cons==0.4.5
contourpy==1.0.7
cookiecutter==2.1.1
coverage==6.5.0
cramjam==2.6.2
crashtest==0.4.1
cryptography==37.0.4
cvxopt==1.3.0
cvxpy==1.3.1
cycler==0.11.0
cymem==2.0.7
Cython==3.0.2
cytoolz==0.12.1
daqp==0.5.1
dash==2.11.1
dash-core-components==2.0.0
dash-html-components==2.0.0
dash-table==5.0.0
dask==2023.8.0
dask-glm==0.2.0
dask-image==2023.8.1
dask-ml==2023.3.24
dataclasses-json==0.5.7
datasette==0.64.3
datasette-graphql==2.2
datashader==0.15.2
datashape==0.5.2
db.py==0.5.4b1
deap==1.3.1
debugpy==1.8.0
decorator==4.4.2
defusedxml==0.7.1
Deprecated==1.2.13
deprecation==2.1.0
diff-match-patch==20200713
dill==0.3.5
dirty-cat==0.4.1
distlib==0.3.6
distributed==2023.8.0
Django==4.1.2
dnspython==2.1.0
docopt==0.6.2
docstring-to-markdown==0.10
docutils==0.18.1
duckdb==0.8.1
dulwich==0.21.5
ecos==2.0.12
emcee==3.1.2
entrypoints==0.4
et-xmlfile==1.1.0
etuples==0.3.8
executing==1.2.0
fast-histogram==0.11
fastai==2.7.12
fastapi==0.103.1
fastcore==1.5.11
fastdownload==0.0.7
fasteners==0.17.3
fastjsonschema==2.16.2
fastparquet==2023.4.0
fastprogress==1.0.3
feather-format==0.4.1
filelock==3.12.0
filterpy==1.4.5
Fiona==1.9.3
flake8==5.0.4
Flask==2.2.2
Flask-Mail==0.9.1
Flask-Session==0.4.0
Flask-SQLAlchemy==3.0.3
flit==3.8.0
flit_core==3.8.0
folium==0.14.0
fonttools==4.37.4
formlayout==1.2.1a1
fqdn==1.5.1
frozenlist==1.3.0
fsspec==2023.5.0
future==0.18.2
fuzzywuzzy==0.18.0
gast==0.4.0
GDAL==3.6.4
geographiclib==1.52
geopandas==0.14.0
geopy==2.2.0
gitdb==4.0.10
GitPython==3.1.32
gmpy2==2.1.5
gpt4all==1.0.8
gpytorch==1.10
graphene==3.2.2
graphql-core==3.2.3
graphql-relay==3.2.0
greenlet==2.0.2
guidata==3.0.4
guiqwt==4.4.1
h11==0.12.0
h2==4.1.0
h5netcdf==1.1.0
h5py==3.7.0
HeapDict==1.0.1
holoviews==1.17.1
hpack==4.0.0
html5lib==1.1
httpcore==0.15.0
httpie==3.2.1
httpx==0.23.0
huggingface-hub==0.17.1
hupper==1.10.3
hvplot==0.8.4
hypercorn==0.14.3
hyperframe==6.0.1
hypothesis==6.75.2
idlex==1.22
idna==3.1
imageio==2.31.1
imageio-ffmpeg==0.4.8
imagesize==1.4.1
imbalanced-learn==0.10.1
importlib-metadata==6.8.0
inflection==0.5.1
iniconfig==1.1.1
installer==0.7.0
intervaltree==3.0.2
ipycanvas==0.13.1
ipydatagrid==1.2.0
ipykernel==6.25.2
ipyleaflet==0.17.4
ipympl==0.9.3
ipython==8.15.0
ipython-genutils==0.2.0
ipython-sql==0.5.0
ipywidgets==8.1.1
isoduration==20.11.0
isort==5.10.1
itsdangerous==2.1.2
janus==1.0.0
jaraco.classes==3.2.3
jedi==0.18.2
jellyfish==0.11.2
Jinja2==3.1.2
jinja2-time==0.2.0
joblib==1.3.1
json5==0.9.8
jsonpointer==2.3
jsonschema==4.17.3
julia==0.6.1
jupyter==1.0.0
jupyter-bokeh==3.0.7
jupyter-console==6.4.4
jupyter-events==0.6.3
jupyter-lsp==2.2.0
jupyter-server-mathjax==0.2.6
jupyter-sphinx==0.4.0
jupyter_client==8.2.0
jupyter_core==5.3.0
jupyter_packaging==0.12.3
jupyter_server==2.5.0
jupyter_server_terminals==0.4.4
jupyterlab==4.0.6
jupyterlab-pygments==0.2.2
jupyterlab-widgets==3.0.9
jupyterlab_server==2.24.0
keyring==23.13.1
kiwisolver==1.4.3
langchain==0.0.292
langcodes==3.3.0
langsmith==0.0.37
lazy-object-proxy==1.7.1
lazy_loader==0.2
linear-operator==0.4.0
linkify-it-py==2.0.0
llm==0.10
llm-gpt4all==0.1.1
llm-llama-cpp==0.1a0
llm-markov==0.1
llvmlite==0.41.0
lmfit==1.0.3
locket==1.0.0
lockfile==0.12.2
logical-unification==0.4.5
loky==3.4.0
lxml==4.9.0
lz4==4.3.2
Mako==1.2.0
Markdown==3.3.7
markdown-it-py==2.2.0
MarkupSafe==2.1.1
marshmallow==3.12.1
marshmallow-enum==1.5.1
matplotlib==3.8.0
matplotlib-inline==0.1.6
maturin==1.2.3
mccabe==0.7.0
mdit-py-plugins==0.3.5
mdurl==0.1.2
mercantile==1.2.1
mergedeep==1.3.4
miniKanren==1.0.3
missingno==0.5.1
mistune==2.0.5
mizani==0.9.2
mlxtend==0.22.0
more-itertools==9.0.0
moviepy==1.0.3
mpl-scatter-density==0.7
mpld3==0.5.8
mpmath==1.2.1
msal==1.22.0
msal-extensions==1.0.0
msgpack==1.0.4
msvc-runtime==14.34.31931
multidict==6.0.2
multipledispatch==0.6.0
munch==2.5.0
murmurhash==1.0.9
mutagen==1.46.0
mypy==1.5.1
mypy-extensions==1.0.0
mysql-connector-python==8.0.21
nbclient==0.8.0
nbconvert==7.8.0
nbdime==3.1.1
nbformat==5.7.0
nbval==0.9.6
nest-asyncio==1.5.6
netCDF4==1.6.0
networkx==3.1
nlopt==2.7.1
nltk==3.8.1
notebook==7.0.4
notebook_shim==0.2.3
ntlm-auth==1.5.0
numba==0.58.0
numcodecs==0.10.0
numdifftools==0.9.40
numexpr==2.8.6
numpy==1.25.2
numpydoc==1.3
openai==0.28.0
opencv-python==4.8.0.74
openpyxl==3.0.10
opt-einsum==3.3.0
orjson==3.7.3
osqp==0.6.2.post9
outcome==1.2.0
packaging==22.0
pandas==2.1.1
pandocfilters==1.5.0
panel==1.2.3
papermill==2.4.0
param==1.13.0
paramiko==2.8.0
parso==0.8.3
partd==1.4.0
pathspec==0.11.0
pathy==0.10.2
patsy==0.5.3
pdfrw==0.4.post2
pep8==1.7.1
pexpect==4.8.0
pg8000==1.23.0
pickleshare==0.7.5
Pillow==9.5.0
PIMS==0.6.1
Pint==0.19.2
pkginfo==1.9.6
platformdirs==3.8.1
plotly==5.16.1
plotnine==0.12.3
pluggy==1.0.0
plumbum==1.8.2
ply==3.11
pmdarima==2.0.3
poetry==1.5.1
poetry-core==1.6.1
poetry-plugin-export==1.5.0
polars==0.19.3
portalocker==2.7.0
portpicker==1.5.0
ppci==0.5.9
preshed==3.0.8
prettytable==3.3.0
priority==2.0.0
proglog==0.1.10
prometheus-client==0.15.0
prompt-toolkit==3.0.39
protobuf==4.23.2
psutil==5.9.5
ptpython==3.0.23
ptyprocess==0.7.0
PuLP==2.6.0
pure-eval==0.2.2
py2vega==0.6.1
pyaedt==0.7.3
pyaml==20.4.0
pyarrow==13.0.0
PyAudio==0.2.11
pybars3==0.9.7
pybind11==2.11.1
pycodestyle==2.9.1
pycosat==0.6.3
pycparser==2.21
pycryptodomex==3.18.0
pyct==0.4.8
pydantic==2.3.0
pydantic_core==2.6.3
pydeck==0.8.0
pydocstyle==6.3.0
pydub==0.25.1
pyepsg==0.4.0
pyerfa==2.0.0.1
pyflakes==2.5.0
pyflux==0.4.17
pygad==3.2.0
pygame==2.4.0
Pygments==2.15.1
PyJWT==2.4.0
pylint==2.15.4
pylint-venv==3.0.2
pyls-spyder==0.4.0
pymc==5.5.0
PyMeta3==0.5.1
pymongo==4.3.3
Pympler==1.0.1
PyNaCl==1.5.0
pynndescent==0.5.7
pyodbc==4.0.39
PyOpenGL==3.1.6
pypandoc==1.5
pyparsing==3.0.9
pypdf==3.15.2
pypng==0.20220715.0
pyproj==3.4.1
pyproject_hooks==1.0.0
PyQt5==5.15.6
PyQt5-Qt5==5.15.2
PyQt5-sip==12.11.0
pyqtgraph==0.13.3
PyQtWebEngine==5.15.5
PyQtWebEngine-Qt5==5.15.2
pyro-api==0.1.2
pyro-ppl==1.8.4
pyrsistent==0.18.1
pyserial==3.5
PySocks==1.7.1
pystache==0.5.4
pytensor==2.12.3
pytest==7.2.1
python-barcode==0.15.1
python-dateutil==2.8.2
python-dotenv==1.0.0
python-json-logger==2.0.7
python-lsp-black==1.3.0
python-lsp-jsonrpc==1.1.1
python-lsp-server==1.7.4
python-multipart==0.0.5
python-slugify==6.1.2
python-snappy==0.6.1
python-ulid==1.1.0
pythonnet==3.0.2
PythonQwt==0.10.2
pytoolconfig==1.2.4
pytz==2023.3
pytz-deprecation-shim==0.1.0.post0
pyviz_comms==3.0.0
PyWavelets==1.4.1
pywin32==306
pywin32-ctypes==0.2.0
pywinpty==2.0.9
pywinusb==0.4.2
PyYAML==6.0
pyzmq==25.1.1
pyzo==4.12.7
QDarkStyle==3.1
qdldl==0.1.7
qpsolvers==3.4.0
qrcode==7.4.2
qstylizer==0.2.2
QtAwesome==1.2.3
qtconsole==5.4.4
QtPy==2.4.0
quadprog==0.1.11
quantecon==0.5.3
quart==0.18.3
rapidfuzz==2.13.7
rasterio==1.3.6
readme-renderer==35.0
redis==4.3.1
regex==2022.9.13
reportlab==4.0.4
requests==2.31.0
requests-ntlm==1.1.0
requests-toolbelt==0.10.1
retrying==1.3.4
rfc3339-validator==0.1.4
rfc3986==1.5.0
rfc3986-validator==0.1.1
rich==13.4.2
rope==1.6.0
rpyc==5.3.1
Rtree==1.0.0
Rx==3.1.1
safetensors==0.3.3
scikit-image==0.21.0
scikit-learn==1.3.1
scikit-optimize==0.9.0
scikit-rf==0.29.1
scipy==1.11.1
scramp==1.4.1
scs==3.2.3
seaborn==0.13.0.dev0
Send2Trash==1.8.0
shapely==2.0.1
shellingham==1.5.0.post1
simplegeneric==0.8.1
simplejson==3.17.6
simpy==4.0.1
six==1.16.0
slicerator==1.1.0
smart-open==5.2.1
smmap==5.0.0
snakeviz==2.1.0
sniffio==1.2.0
snowballstemmer==2.2.0
snuggs==1.4.7
sortedcontainers==2.4.0
sounddevice==0.4.4
soupsieve==2.3.2.post1
spacy==3.6.1
spacy-legacy==3.0.12
spacy-loggers==1.0.3
Sphinx==6.1.3
sphinx-rtd-theme==1.2.0
sphinxcontrib-applehelp==1.0.2
sphinxcontrib-devhelp==1.0.2
sphinxcontrib-htmlhelp==2.0.0
sphinxcontrib-jquery==4.1
sphinxcontrib-jsmath==1.0.1
sphinxcontrib-qthelp==1.0.3
sphinxcontrib-serializinghtml==1.1.5
spyder==5.4.5
spyder-kernels==2.4.4
SQLAlchemy==2.0.19
sqlite-bro==0.12.2
sqlite-fts4==1.0.3
sqlite-migrate==0.1a2
sqlite-utils==3.35.1
sqlparse==0.4.3
srsly==2.4.5
sspyrs==0.3
stack-data==0.6.1
starlette==0.27.0
statsmodels==0.14.0
streamlit==1.23.1
streamz==0.6.3
supersmoother==0.4
swifter==1.3.4
sympy==1.12
tables==3.7.0
tabulate==0.9.0
tbats==1.1.0
tblib==1.7.0
tenacity==8.1.0
terminado==0.17.0
text-unidecode==1.3
textdistance==4.5.0
textwrap3==0.9.2
thinc==8.1.12
threadpoolctl==3.1.0
three-merge==0.1.1
tifffile==2023.7.18
tinycss2==1.1.1
tokenizers==0.13.3
toml==0.10.2
tomli==2.0.1
tomli_w==1.0.0
tomlkit==0.11.6
toolz==0.12.0
torch==2.0.1
torchaudio==2.0.2
torchvision==0.15.2
tornado==6.3.1
tqdm==4.65.0
traitlets==5.10.0
traittypes==0.2.1
transformers==4.33.2
trio==0.22.2
trio-asyncio==0.12.0
trove-classifiers==2023.2.20
twine==4.0.1
typer==0.9.0
typing-inspect==0.8.0
typing_extensions==4.7.1
tzdata==2022.7
tzlocal==4.2
uc-micro-py==1.0.1
ujson==5.3.0
umap-learn==0.5.1
uncertainties==3.1.6
uri-template==1.2.0
urllib3==1.26.10
uvicorn==0.22.0
validators==0.18.2
vega-datasets==0.9.0
virtualenv==20.23.0
waitress==2.1.2
wasabi==0.10.1
wasmer==1.1.0
wasmer-compiler-cranelift==1.1.0
wasmer-compiler-singlepass==1.1.0
watchdog==2.2.0
wcwidth==0.2.5
webcolors==1.12
webencodings==0.5.1
websocket-client==1.6.1
websockets==10.3
Werkzeug==2.2.2
whatthepatch==1.0.2
widgetsnbextension==4.0.9
winpython==7.0.20230923
wordcloud==1.8.1
wrapt==1.14.1
wsproto==1.2.0
xarray==2023.8.0
xarray-einstats==0.5.1
xgboost==1.6.1
XlsxWriter==3.1.2
xlwings==0.24.7
xmltodict==0.13.0
xyzservices==2022.6.0
yapf==0.40.1
yarl==1.7.2
yt-dlp==2023.7.6
zarr==2.11.3
zict==2.2.0
zipp==3.9.0
zstandard==0.21.0

**Comment by ring630** [comment]
@svandenb-dev can you have a look at this issue?

**Comment by svandenb-dev** [comment]
@askoeho Yes this seems to be a bug. However I think setting create_circuit_port=True should work ? 
This is a recommended setting to prevent shorts in case the port intersect metal. Unless you want to export to HFSS 3D modeler.
Will have a look. Thanks for your feedback

**Comment by askoeho** [comment]
create_circuit_port=True works as intended, circuit ports are created. But these are not 1:1 with gap ports and at the same time I would like to have the export to HFSS 3D modeler working.

**Comment by svandenb-dev** [comment]
@askoeho bug is fixed please grab the latest release when it's available.
